frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -5,6 +5,14 @@
     </div>
     <div class="card-body">
       <form>
+        <div class="form-group">
+          <label>{{ T.userEditBirthDate }}</label>
+          <omegaup-datepicker
+            v-model="birthDate"
+            :required="false"
+            :max="new Date()"
+          ></omegaup-datepicker>
```

Comment:
In the tests you are trying access to this element by `data-attributes` and `name`, so you need add both of them:
 
```suggestion
          <omegaup-datepicker
            name="reg_birthdate"
            data-signup-birthdate
            v-model="birthDate"
            :required="false"
            :max="new Date()"
          ></omegaup-datepicker>
```